### PR TITLE
Add branch cache to version and fetch storage data from server metadata

### PR DESCRIPTION
**Pull request by shifucun**
Instead of fetching version from environment, save them in server metadata and use from that.

Turns out branch cache was disabled in a previous PR. Fixed that here too.

### CODE REVIEW

File: proto/misc.proto [hunk 1]
```diff
@@ -72,8 +72,12 @@ message GetVersionResponse {
   string store = 1;
   // Big Query dataset
   string big_query = 2;
-  // Bigtable instance
-  string big_table = 3;
+  // Base Bigtable instance
```
Comment (pradh) [inline proto/misc.proto:75]: needs an aligned website change?

File: cmd/main.go [hunk 2]
```diff
@@ -53,7 +52,7 @@ var (
 	useBaseBt     = flag.Bool("use_base_bt", true, "Use base bigtable cache")
 	baseTableName = flag.String("base_table", "", "Base cache Bigtable table.")
 	// Branch Bigtable Cache
-	useBranchBt = flag.Bool("use_branch_bt", false, "Use branch bigtable cache")
+	useBranchBt = flag.Bool("use_branch_bt", true, "Use branch bigtable cache")
```
Comment (pradh) [inline cmd/main.go:55]: how do we catch such regressions?

**Review by pradh — APPROVED** [review]

File: proto/misc.proto (hunk 1 above)
Comment (shifucun) [reply proto/misc.proto:75]: Yes, website will use this API for versioning too.

File: cmd/main.go (hunk 2 above)
Comment (shifucun) [reply cmd/main.go:55]: We should have a prober to fetch the few stat vars from branch cache and make sure the latest date is no more than N days.

**Pull request merged**